test/models/subject_test.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+class NotificationTest < ActiveSupport::TestCase
+  test 'author_url returns a user url with users' do
+    subject = create(:subject, author: "Fryguy")
+    assert_equal "https://github.com/Fryguy", subject.author_url
+  end
+
+  test 'author_url returns a GitHub apps URL with bots' do
+    subject = create(:subject, author: "greenfield[bot]")
+    assert_equal "https://github.com/apps/greenfield", subject.author_url
```

Comment:
Invalid link - gives `HTTP 404 - Not found`. It's seems to be a `greenkeeper`

Reply:
Hah, I forgot the name already. Fortunately this is just test so it doesn't really matter ^_^ I'll change it anyway, thanks.